input/fsh/DEF_Rulesets.fsh

```diff
@@ -189,6 +194,9 @@ RuleSet: RadiotherapyPhaseAndPlanPrescriptionCommon
   * extension[fractions] 0..0
   * extension[fractions] ^short = "Not used in this profile. In a Treatment Plan, all volumes are involved in all fractions."
   * extension[fractions] ^definition = "Not used in this profile. In a Treatment Plan, all volumes are involved in all fractions and the number of fractions is defined in extension radiotherapyFractionsPrescribed."
+  * extension[totalDose] ^short = "Total Physical Radiation Dose Prescribed"
```

Comment:
The Phyiscal Dase should be in the extensions radiotherapy-dose-prescribed-to-volume and radiotherapy-dose-planned-to-volume. It it applies wherever the two extensions are used, i.e. also to PlannedCourse and Course Prescription. And where we do not define the extension ourselves, but it comes from mCODE, there we can rely on mCODE adding it, or add such a ^short to ProceduresCommon (applying to CourseSummary, TreatedPhase, TreatedPlan).

Reply:
It is included in:
* RadiotherapyDosePrescribedToVolume
* RadiotherapyDosePlannedToVolume
* RadiotherapyFractionsDelivered
* RadiotherapyProcedureCommon has: * extension[doseDeliveredToVolume].extension[totalDoseDelivered] ^short = "Total Physical Radiation Dose Delivered"

Does that cover it?